.github/workflows/ci.yml

```diff
@@ -10,8 +10,14 @@ jobs:
 
```

Comment:
can we also add a CI for windows?

Reply:
Yes, I am working on it. I just installed a windows system in my PC. I'll try this out these days.